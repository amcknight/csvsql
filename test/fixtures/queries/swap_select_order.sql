SELECT last_name, first_name
FROM test/fixtures/tables/people
WHERE age > 30
