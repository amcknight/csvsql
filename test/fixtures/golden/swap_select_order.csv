last_name, first_name
McKnight, Andrew
Ash, Tom
Carlson, Cayley
